lib/srv/db/sqlserver/protocol/packet.go

```diff
@@ -158,3 +173,8 @@ func makePacket(pktType uint8, pktData []byte) ([]byte, error) {
 
 	return append(headerBytes, pktData...), nil
 }
+
+// IsFinalPacket returns true there are no more packets on the message.
+func IsFinalPacket(packet Packet) bool {
+	return packet.Header().Status&PacketStatusLast != 0
+}
```

Comment:
Can we do an equal `==` comparison instead?
There is no other status which has its last bit as `1`, so, I believe this is still correct.
Only after you look up docs, can you assume that's true.
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/ce398f9a-7d47-4ede-8f36-9dd6fc21ca43

Using the `==` comparison, IMO, would make the code simpler.

Unless this is as is because of some other reason, ofc.
eg, Is this a performance optimisation?

Reply:
There is no limitation on comparing with `== 1`. I've only used the same logic from the driver (which does a similar check). But I agree it looks easier to understand, and will update it.